app/src/main/java/com/github/campus_capture/bootcamp/firebase/PlaceholderBackend.java

```diff
@@ -25,7 +40,23 @@ public CompletableFuture<Boolean> hasAttacked(String uid) {
     @Override
     public CompletableFuture<Map<String, Section>> getCurrentZoneOwners() {
         Map<String, Section> out = new HashMap<>();
-        out.put("campus", Section.IN);
+        out.put("SG1", AR);
```

Comment:
Nice centralization! This is cleaner than recreate a mock backend each time! Good!